modules/claims_api/app/workers/claims_api/claim_establisher.rb

```diff
@@ -68,18 +69,9 @@ def service(auth_headers)
     end
 
     def bgs_user(auth_headers)
-      user = OpenStruct.new(ssn: auth_headers['va_eauth_pnid'],
-                            uuid: nil,
-                            email: nil,
-                            icn: nil,
-                            common_name: nil)
-      return user if auth_headers['va_bgs_authorization'].blank?
-
-      bgs_auth_headers = JSON.parse(auth_headers['va_bgs_authorization'])
-      user.uuid = bgs_auth_headers['external_uid']
-      user.email = bgs_auth_headers['external_key']
-
-      user
+      {
+        'ssn' => auth_headers['va_eauth_pnid']
```

Comment:
pretty sure you just want to take the participant_id off the headers stored with the claim record and add it to this user object here, thats already getting passed into subsequent jobs